rust/src/conf.rs

```diff
@@ -103,29 +101,6 @@ impl ConfNode {
         return Self { conf }
     }
 
-    pub fn get_child_value(&self, key: &str) -> Option<&str> {
```

Comment:
I consider this part of the public api footprint.

Reply:
ok